src/main/java/edu/wpi/grip/ui/ExceptionView.java

```diff
@@ -0,0 +1,191 @@
+package edu.wpi.grip.ui;
+
+import javafx.application.HostServices;
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.scene.control.*;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Priority;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Displays an alert with an exception and provides a place for a user to supply information about what caused the error.
+ * Includes a textbox with formatted markdown to allow the issue to be pasted into GitHub easily.
+ * Also, provides links with quick access to the githup issue page.
+ */
+public final class ExceptionView extends Alert {
+    private static final String PROJECT_ISSUE_LINK = "https://github.com/WPIRoboticsProjects/GRIP/issues";
+    private static final String ISSUE_PROMPT_QUESTION = "What were the actions that caused this error occurred?";
+
+    private static final String systemOptions[] = {
```

Comment:
I'm not sure where this list came from, but I think the properties that might be useful are `os.name`, `os.version`, `os.arch`, `java.vm.name`, `java.version`, and `javafx.version`.